.tekton/konflux-e2e-tests-push.yaml

```diff
@@ -433,6 +433,9 @@ spec:
         - name: kind
           value: task
         resolver: bundles
+      workspaces:
+      - name: workspace
+        workspace: workspace
```

Comment:
just curious, where does this change come from?

Reply:
🥇 I broke the pipelinerun in one of my previous commits 🥇 

All push pipelines are failing due this missing workspace

Reply:
nice! 🏆